frontend/app/components/routing/wp-list/wp-list.controller.ts

```diff
@@ -159,6 +159,18 @@ function WorkPackagesListController($scope,
     AuthorisationService.initModelAuth("query", meta.query._links);
   }
 
+  $scope.setAnchorToNextPage = function () {
+    anchor = '';
+    if(jQuery('.pagination-number').last().hasClass('-current')) {
```

Comment:
@oliverguenther is there a more elegant way to find out wether the last pagination element is reached?